kategory-test/src/main/kotlin/kategory/laws/MonadFilterLaws.kt

```diff
@@ -8,15 +8,17 @@ object MonadFilterLaws {
     inline fun <reified F> laws(MF: MonadFilter<F> = monadFilter<F>(), crossinline cf: (Int) -> HK<F, Int>, EQ: Eq<HK<F, Int>>): List<Law> =
             MonadLaws.laws(MF, EQ) + FunctorFilterLaws.laws(MF, cf, EQ) + listOf(
                     Law("MonadFilter Laws: Left Empty", { monadFilterLeftEmpty(MF, EQ) }),
-                    Law("MonadFilter Laws: Right Empty", { monadFilterRightEmpty(MF, cf, EQ) }),
-                    Law("MonadFilter Laws: Consistency", { monadFilterConsistency(MF, cf, EQ) }))
+                    Law("MonadFilter Laws: Right Empty", { monadFilterRightEmpty(MF, EQ) }),
+                    Law("MonadFilter Laws: Consistency", { monadFilterConsistency(MF, cf, EQ) }),
```

Comment:
Can we use applicative pure instead of constructor function?

Reply:
I just removed that `cf` param because it was not used but did not added anything to that in this PR. But presumably yes.